src/app/Components/thumbnailConcert.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Navigation } from 'react-router';
+import { connect } from 'react-redux';
+import { currentDisplayedGift } from '../Actions/gift';
+
+var ThumbnailConcert = React.createClass({
```

Comment:
Note: might want to rename this to ConcertDetails bc it includes more than just the thumbnail